Apps/Src/SendTritium.c

```diff
@@ -388,7 +389,12 @@ static void ReverseDriveHandler(){
         UpdateDisplay_SetGear(DISP_REVERSE);
     }
     velocitySetpoint = -MAX_VELOCITY;
-    currentSetpoint = percentToFloat(map(accelPedalPercent, ACCEL_PEDAL_THRESHOLD, PEDAL_MAX, CURRENT_SP_MIN, CURRENT_SP_MAX));
+    if (velocityObserved >= MAX_REVERSE_VELOCITY) {
+        currentSetpoint = percentToFloat(map(accelPedalPercent, ACCEL_PEDAL_THRESHOLD, PEDAL_MAX, CURRENT_SP_MIN, CURRENT_SP_MAX * 0));  // cutoff current to motor
```

Comment:
No point replicating this twice, just assign some variable to the current.

Reply:
Changed.